src/App.tsx

```diff
@@ -12,6 +12,14 @@ export class App extends Component<{}, State> {
     movies: [],
   };
 
+  addMovies = (movie:Movie) => {
+    if (this.state.movies.every(el => el.imdbID !== movie.imdbID)) {
+      this.setState(state => ({
+        movies: [...state.movies, movie],
+      }));
+    }
```

Comment:
Here you should find only 1 element that exists, better use some method

Reply:
Agree, use `some` here